src/KvkClient.php

```diff
@@ -11,7 +11,7 @@
 use Werkspot\KvkApi\Http\Endpoint\MapperInterface;
 use Werkspot\KvkApi\Http\Search\ProfileQuery;
 
-final class KvkClient
+class KvkClient
```

Comment:
I would rather create an interface for this, even though it is not necessary for this project itself.
The reason the `final` keyword is there is to prevent client projects from extending it, which would make breaking changes easier to happen and could become a problem when we would need to change this class.

Reply:
You're correct, so let's introduce an interface for it.